lib/websocket_server.rb

```diff
@@ -2,6 +2,8 @@ class WebsocketServer
   Pairing = Struct.new(:is_ws, :proxy)
 
   def initialize
+    @logger = Rails.env.production? ? $websocket_log : Logger.new(STDOUT)
```

Comment:
Do you run this script standalone?  Maybe you want to create a stdout logger if this script is executed directly

by using something like this:
`if __FILE__ == $PROGRAM_NAME`